packages/integrations/vercel/src/serverless/adapter.ts

```diff
@@ -439,6 +520,20 @@ async function createFunctionFolder({
 		maxDuration,
 		supportsResponseStreaming: true,
 	});
+	
+	if (isr) {
+		const {
+			revalidate: expiration = false,
+			bypassToken = undefined,
+		} = typeof isr === "object" ? isr : {};
```

Comment:
The extraction of the fields here can be cleaner, especially considering that we do `typeof isr === "object"` every single time we need to read `isr`

Reply:
Cleaned up. [Here is the new control flow](https://github.com/withastro/astro/pull/9714/files#diff-071ccfbdfd7b9bc803219179f88ae0bdb227a399bc4a481b7831c82b801d7a17R487-R497)